main/tests/data/big5.html

```diff
@@ -13,11 +13,11 @@
         </div>
       </div>
       <div class="COM">
-        <p align="center" style="font-size:16px;">�N���z�ɦ^�i��޵��L��j�x��</p>
+        <p align="center" style="font-size:16px;">�N���z�ɦ^�i��޵��L���j�x��</p>
```

Comment:
This file should not have been touched at all, this is a test file for encoding support at the import stage (it is not used in OpenRefine's UI). By editing it you are likely changing the way the special characters are encoded, so changing the meaning of the associated test. You should be able to fix this by reverting this file (and others in the `main/tests/data` directory) to the version before your changes.

Reply:
@optimm it looks like this problem has not been solved yet